core/src/main/java/org/apache/carbondata/core/datastore/page/statistics/ColumnPageStatistics.java

```diff
@@ -33,30 +34,30 @@
    * the unique value is the non-exist value in the row,
    * and will be used as storage key for null values of measures
    */
-  private Object uniqueValue;
+  private Object nonExistValue;
```

Comment:
But it will be written into file by `CarbonMetadataUtil.serializeEncodeMetaUsingByteBuffer`, you mean read path is not using it?